internal/experimental/experimental.go

```diff
@@ -0,0 +1,75 @@
+package experimental
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	envName = "NOTATION_EXPERIMENTAL"
+	enabled = "1"
+)
+
+// IsDisabled determines whether an experimental feature is disabled.
+func IsDisabled() bool {
+	return os.Getenv(envName) != enabled
+}
+
+// Error returns an error for a disabled experimental feature.
+func Error(description string) error {
+	return fmt.Errorf("%s been marked as experimental and not enabled by default. To use it, please set %s=%s in your environment", description, envName, enabled)
```

Comment:
```suggestion
	return fmt.Errorf("%s is experimental and not enabled by default. To use, please set %s=%s environment variable", description, envName, enabled)
```

Reply:
Changed accordingly. There is a case when multiple experimental flags the warning can be
> flag(s) --foo,--bar,--baz in "notation test" is experimental and not enabled by default. To use, please set %s=%s environment variable

I am okay with the `is` here, @patrickzheng200 what do you think.

Reply:
> Changed accordingly. There is a case when multiple experimental flags the warning can be
> 
> > flag(s) --foo,--bar,--baz in "notation test" is experimental and not enabled by default. To use, please set %s=%s environment variable
> 
> I am okay with the `is` here, @patrickzheng200 what do you think.

Yeah, we could have multiple flags marked as experimental, in this case, using `is` here is not very accurate.